docs/contributor-guide/developer-guide/feature-flags.md

```diff
@@ -114,3 +114,20 @@ in the existing tests until you also update them.
+
+## Releasing a Feature Behind a Feature Flag
+
+The following process should be used to implement a feature flag, release it in a build, and then remove it once the feature has been used in production.
+
+1. Create a Github issue using the Feature Flag template that covers the creation and deletion of the feature flag that will guard your feature. Ensure it is tied to the appropriate epic so that the issue must be completed before the epic is considered complete.
+2. Create the feature flag in code ([example](https://github.com/civiform/civiform/pull/4435)), update the issue to mention that the feature flag now exists. Enable the feature flag in [dev](https://github.com/civiform/civiform/blob/main/server/conf/application.dev.conf). Disable the flag for [browser tests](https://github.com/civiform/civiform/blob/main/server/conf/application.dev-browser-tests.conf), since the browser tests will turn the flag on and off inside the tests themselves.  For the PR that adds the flag, ensure the release notes section describes what this flag is for.
+3. Write your feature and guard it with the flag.
+4. Pass the flag through the deployment system to allow enabling it in staging ([example](https://github.com/civiform/cloud-deploy-infra/commit/9d17356ff1fa1f3a16c97608cc00cbd4c7c11ffe)).
+5. When the next release is created, ensure information about the release flag is in the email sent for the release.
+6. Enable the flag in AWS staging and Azure staging and ask Seattle engineers to enable the flag in their staging environment.
+7. Optional (depending on the complexity of your feature): Organize a bug bash where team members try to break your feature
+8. Send a message to Matthew Sprenke in the #release-discussion channel on CiviForm Slack to check if Seattle wants to do some manual QA.
+9. Set the default value of the flag to `true` to make it default enabled (e.g. [changing this to true](https://sourcegraph.com/github.com/civiform/civiform/-/blob/server/conf/helper/feature-flags.conf?L27)). In the PR, include release notes, which mention that the specific feature flag's default value was changed and what feature it enables. Update the feature flag Github issue to note when the default was changed. Assign the issue to yourself or someone who will be around in the future.
+10. Ensure that the feature is enabled in an actual production build, because each deployment can choose to enable or disable the flags of their choosing (often the easiest deployment to use will be in Seattle) and note the date when it was enabled in the Github issue.
+11. Leave the flag in the codebase for at least another month after it went live in production (e.g. Seattle).
+12. Remove the flag, remove the guards in the code looking for the flag, note in the PR release notes that the flag is being removed, then close the Github issue. Ensure the email sent for the next release includes a note about the removal of the feature flag.
```

Comment:
**thought (non-blocking):** over time we'll probably see a decent number of feature flags left over from fellows who rotated off before it was time for removel